tools/onnx-subgraph/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# cmake version dependency
```

Comment:
plz rename folder `onnx-subgraph` to `onnx_subgraph`.
all other names uses `_`, not `-`